<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Mux Switcher</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of sweeps per electrode</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushRun">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Test</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Mask</string>
      </property>
     </widget>
     <widget class="QLabel" name="switchNotice">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="maskOptionSwitcher">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>20</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="automaticallyParse">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatically parse Radiant output data</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numSweeps">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Data Parser</string>
     </attribute>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Raw result folder:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rawResults">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>20</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>90</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ex. C:/Users/YouPutItHere</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="folderName">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>130</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>130</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ex. Sample Folder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>90</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Put results in:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="filePath">
      <property name="geometry">
       <rect>
        <x>750</x>
        <y>90</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>130</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Result folder name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>50</y>
        <width>571</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Please configure to name raw result files &quot;group1_data.txt&quot;, &quot;group2_data.txt&quot;, &quot;group3_data.txt&quot;, &quot;group4_data.txt&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>20</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ex. C:/Users/RadiantOutput</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mask</string>
      </property>
     </widget>
     <widget class="QPushButton" name="runParser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>351</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Run Data Parser</string>
      </property>
     </widget>
     <widget class="QLabel" name="parserNotice">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="maskOptionParser">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string/>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Number of sweeps per electrode</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="numSweeps_parser">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <property name="value">
       <number>3</number>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun_Test">
    <property name="title">
     <string>Mux-Radiant Control</string>
    </property>
   </widget>
   <addaction name="menuRun_Test"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
